src/esp/bindings_js/bindings_js.cpp

```diff
@@ -218,6 +262,7 @@ EMSCRIPTEN_BINDINGS(habitat_sim_bindings_js) {
                                    &Agent::getSensorSuite))
       .function("getState", &Agent::getState)
       .function("setState", &Agent::setState)
+      .function("setRotation", &Agent_setRotation)
```

Comment:
So for consistency, rotation is a part of AgentState on the Python side Is it not the on C++ side, @bigbike ?